src/write/pages.js

```diff
@@ -46,9 +40,9 @@ function writePages (pages, drizzleData, writePromises = []) {
  * TODO: Better resolve/return value
  */
 function pages (drizzleData) {
-  return Promise.all(writePages(drizzleData.context.pages, drizzleData))
+  return Promise.all(writePages(drizzleData.pages, drizzleData))
     .then(() => {
-      return true;
+      return drizzleData;
```

Comment:
So to be clear, the promises mutate `drizzleData` and then we return the resulting, changed object?